### DISCUSSION TITLE
Keep unread articles forever, remove only read

### DISCUSSION THREAD

**Discussion by EekoS** [FreshRSS/FreshRSS#5857] [Q&A] [answered]
Hello! I am new to FreshRSS - and already loving it!

I would like to keep all unread articles forever (both unread and available for me to read even 10 years from now), and remove/purge all the others (I don't really care after how much time - let's say 1 month).

Do the following settings do the trick? 
- Unchecked "Maximum (number+age) of articles to keep..."
- Checked "Never delete unread articles"

Or will these settings keep all articles, regardless if they are unread or read? ...So for my goal, should I set like this?
 - Check "Maximum age of articles to keep 1 month"
 - Check "Never delete unread articles"

This is what I set now:
![image](https://github.com/FreshRSS/FreshRSS/assets/25973161/1178b24c-69ef-4324-824d-dc521b1cd8f8)

**Comment by Alkarex** [comment, marked as answer]
What you have now will keep everything.
To remove read articles, you need to activate one or both of the "maximum" policies.
Your suggestion looks good
